manifests/cluster-infra/5a_alertmanager_deployment.yaml

```
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-config
+data:
+  alertmanager.yml: |
+    global:
+      resolve_timeout: 5m
+
+    route:
+      group_by: ['alertname', 'namespace']
+      group_wait: 1m
+      group_interval: 5m
+      repeat_interval: 2d
+      receiver: amgithubnotifier-bridge
```

lets use cammelCase Everywhere
```suggestion
      receiver: amGithubNotifierBridge
```